rct_optimizations/src/rct_optimizations/dh_chain_kinematic_calibration.cpp

```diff
@@ -10,6 +10,7 @@
 namespace rct_optimizations
 {
 
+// TODO: replace; duplicates a member function
```

Comment:
I think we can delete this comment. [The function I think I was referring to](https://github.com/Jmeyer1292/robot_cal_tools/blob/659c0756affde260c842c8f20d20598903a27bb6/rct_optimizations/include/rct_optimizations/dh_chain_kinematic_calibration.h#L149-L163) when I added the comment ended up being substantially different from the one here.